presto-orc/src/main/java/io/prestosql/orc/writer/MapColumnWriter.java

```diff
@@ -199,7 +199,9 @@ public void close()
         ImmutableList.Builder<StreamDataOutput> indexStreams = ImmutableList.builder();
         indexStreams.add(new StreamDataOutput(slice, stream));
         indexStreams.addAll(keyWriter.getIndexStreams(metadataWriter));
+        indexStreams.addAll(keyWriter.getBloomFilters(metadataWriter));
         indexStreams.addAll(valueWriter.getIndexStreams(metadataWriter));
+        indexStreams.addAll(valueWriter.getBloomFilters(metadataWriter));
```

Comment:
how are blooms used for key and value parts here?

Reply:
Bloom indices are created for both key and value. Support has to be provided for reading them.